simulator/mips/mips_instr.h

```diff
@@ -350,20 +350,13 @@ class MIPSInstr
         uint64 get_v_dst() const { return v_dst; }
         auto get_lwrl_mask() const
         {
-            uint32 mask = 0x0;
-            uint32 i = 0;
```

Comment:
I realized that I wrote exhaustive code. I thought that the mask can start with 0 (for example: `mask = 0xFFFF'FF00`

Reply:
That's true for LWL